modules/pipeline/services/crondsvc/crond.go

```diff
@@ -123,8 +178,8 @@ func (s *CrondSvc) crondSnapshot() []string {
 	return logs
 }
 
-func makePipelineCronName(cron spec.PipelineCron) string {
```

Comment:
why adjust format?

Reply:
cron.CronExpr as the name cannot be used as the only task name of cron, channel cannot know before CronExpr